packages/neos-ui-sagas/src/UI/Inspector/index.js

```diff
@@ -38,6 +40,18 @@ export function * inspectorSaga({globalRegistry}) {
             //
             if (nextAction.type === actionTypes.CR.Nodes.FOCUS) {
                 yield put(actions.UI.Inspector.closeSecondaryInspector());
+
+                const focusedNode = yield select(getFocusedNode);
+                if (!$get('isFullyLoaded', focusedNode)) {
```

Comment:
cant we just use opt chaining in 7.3? ^^

Reply:
we can't, sadly

Reply:
okay then thats fine with me ;)